### ISSUE TITLE
[ Key Error: 'activity inbox' ] after python line: [ print pump.me.inbox ] with mediagoblin webfinger

### ISSUE THREAD

**Issue by SapienTech** [xray7224/PyPump#131]
after verifying mediagoblin account in both pypump-shell and manually via command line, above error occurs when trying to access user's inbox. Note the line [print pump.me] does not return an error.

---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/home/user/Documents/2015_summer/MediaGoblin/PyPump/pypump-shell in <module>()
----> 1 print pump.me.inbox

/home/user/Documents/2015_summer/MediaGoblin/PyPump/pypump/models/person.pyc in inbox(self)
    148         if not self.isme:
    149             raise PyPumpException("You can't read other people's inboxes")
--> 150         self._inbox = self._inbox or Inbox(self.links['activity-inbox'], pypump=self._pump)
    151         return self._inbox
    152 

KeyError: 'activity-inbox'

In [2]: print pump.me
SapienTech

**Comment by kabniel** [comment]
Can you post output of ```pump.me.published``` and ```pump.me.links``` ?

**Comment by SapienTech** [comment]
In [1]: print pump.me.published
None

In [2]: print pump.me.links
{'self': 'https://goblin-jeffers.pagekite.me/api/user/SapienTech/profile'}

**Comment by kabniel** [comment]
[1] tells me that PyPump fails to load the user profile from server

if you run ```$ pypump-shell SapienTech@goblin-jeffers.pagekite.me --log=debug``` then ```pump.me```

you'll probably see ```DEBUG:requests.packages.urllib3.connectionpool:"GET /api/user/SapienTech/profile HTTP/1.1" 400 38``` on the last row, not sure why you get a http error 400 though..

**Comment by SapienTech** [comment]
yes I am getting that. I set up mediagoblin following the instructions on https://wiki.mediagoblin.org/HackingHowto and am currently running the server with $ ./bin/paster serve paste.ini --reload and $ CELERY_CONFIG_MODULE=mediagoblin.init.celery.from_celery ./bin/celeryd
Should I instead follow the instructions on http://mediagoblin.readthedocs.org/en/stable/ ?

**Comment by kabniel** [comment]
I think the problem might be that you have some upper case letters in your user name, and we are not handling it properly.
It would be helpful if you could try to register another user with only lower case letters in the name, and see if you get the same error with that user in pypump.

**Comment by moggers87** [comment]
@SapienTech any update on this?

**Comment by SapienTech** [comment]
@moggers87 sorry for the delay on this! will get back to you in the coming days
